src/rosdistro/verify.py

```diff
@@ -128,17 +128,29 @@ def _check_files_identical(index, dist_name, loader_function, yaml_url, file_typ
         files = [files]
         yaml_url = [yaml_url]
     for i, f in enumerate(files):
-        identical &= _check_file_identical(f, yaml_url[i], file_type)
+        identical &= _check_file_identical(f, yaml_url[i], file_type, ignore_comments=True)
     return identical
 
 
-def _check_file_identical(dist_file, yaml_url, file_type):
+def _check_file_identical(dist_file, yaml_url, file_type, ignore_comments=False):
     yaml_str = load_url(yaml_url)
     yaml_lines = yaml_str.splitlines()
     dist_file_data = dist_file.get_data()
     dist_file_lines = _to_yaml(dist_file_data).splitlines()
     dist_file_lines[0:0] = _yaml_header_lines(file_type, dist_file_data['version'])
 
+    if ignore_comments:
+        import re
+        yaml_lines_to_remove = []
+        yaml_header = _yaml_header_lines(file_type, dist_file_data['version'])
+        for yaml_line in yaml_lines:
+            if yaml_line in yaml_header:
+                continue
+            elif re.match(r'(^#| +#)', yaml_line):
+                yaml_lines_to_remove.append(yaml_line)
+        for yaml_line_to_remove in yaml_lines_to_remove:
+            yaml_lines.remove(yaml_line_to_remove)
```

Comment:
good point, now that I simplified the regex to not process the header but only the comment lstrip will totally do the job :+1:
fixed in https://github.com/ros-infrastructure/rosdistro/pull/98/commits/e1e84a0282445b8ee74099b9aa9c4b0b374daaf5